package.json

```diff
@@ -24,11 +24,11 @@
     "private::build:backends": "dash-generate-components src/dash-table/dash/DataTable.js dash_table -p package-info.json && cp dash_table_base/** dash_table/ && dash-generate-components src/dash-table/dash/DataTable.js dash_table -p package-info.json --r-prefix 'dash' --r-suggests 'dash' --jl-prefix 'dash'",
     "private::format.ts": "npm run private::lint.ts -- --fix",
     "private::format.prettier": "prettier --config .prettierrc --write \"src/**/*.{js,ts,tsx}\"",
-    "private::format.black": "black --exclude dash_table .",
+    "private::format.black": "black --exclude '/(dash_table|venv|dash_table_base)/' .",
```

Comment:
The exclusion of `dash_table` makes sense because it's mainly autogenerated files, but we should be able to blacken `dash_table_base`, shouldn't we?

In fact in the dash repo we just list the directories we DO want to run black on:
```
"private::format.black": "black dash tests --exclude metadata_test.py"
```
so I'd think here the equivalent would be `black dash_table_base tests`

Reply:
@alexcjohnson Thanks for the suggestion; I agree that's more consistent. In 6408184

Reply:
I excluded `dash_table_base` because I couldn't get black to format it, I got 

```
error: cannot format dash_table_base/__init__.py: 
All done! 💥 💔 💥
1 file failed to reformat.
```

without any indication of what was wrong. After removing a couple commas I managed to get

```
error: cannot format dash_table_base/__init__.py: Cannot parse: 12:12:         file=_sys.stderr
```

so I guessed that the `_` short name in `from __future__ import print_function` was confusing black when parsing `_`-prefixed variables and I removed it in 32b1ceb, which fixed the problem. I'm not sure why there was an alias there, as AFAIK this `print_function` is never explicitly called.

Reply:
Annoying, I've never seen black behave like that, but I'm glad you sorted it out!